imgui-SFML.cpp

```diff
@@ -784,7 +784,11 @@ bool UpdateFontTexture() {
     io.Fonts->GetTexDataAsRGBA32(&pixels, &width, &height);
 
     sf::Texture& texture = s_currWindowCtx->fontTexture;
+#if SFML_VERSION_MAJOR >= 3
+    if (!texture.create(sf::Vector2u(width, height))) {
+#else
```

Comment:
I'd suggest to not have broken off pieces of C++ grammar guarded by conditionals (and do it exactly like https://github.com/eliasdaler/imgui-sfml/issues/207#issuecomment-1129368591)
-this can confuse text editors, particularly easily if curly braces are unbalanced

Reply:
I didn't think of that. I now see the weird highlighting VS Code is giving me.